app/components/github_integration/webhooks/issues.py

```diff
@@ -126,26 +128,41 @@ async def _(event: events.IssuesReopened) -> None:
 
     @webhook.event.issues.edited
     async def _(event: events.IssuesEdited) -> None:
-        issue, changes = event.issue, event.changes
+        issue, old_issue_changes = event.issue, event.changes
 
         if issue.created_at > dt.datetime.now(tz=dt.UTC) - dt.timedelta(minutes=15):
             return
 
-        update_notes: list[str] = []
-        if changes.title:
-            update_notes.append(
-                f'Renamed from "{changes.title.from_}" to "{issue.title}"'
+        if old_issue_changes.body and old_issue_changes.body.from_:
+            # Escape code block to avoid breaking diff
+            from_file = old_issue_changes.body.from_.replace("`", r"\`").splitlines(
+                keepends=True
             )
-        if changes.body:
-            update_notes.append("Updated description")
-
-        match update_notes:
-            case [note]:
-                content = note
-            case [note1, note2]:
-                content = f"* {note1}\n* {note2}"
-            case _:
-                return
+            to_file = (
+                issue.body.replace("`", r"\`").splitlines(keepends=True)
+                if issue.body
+                else ""
+            )
+            diff = "".join(
+                difflib.unified_diff(
+                    from_file,
+                    to_file,
+                    fromfile=old_issue_changes.title.from_
+                    if old_issue_changes.title
+                    else issue.title,
+                    tofile=issue.title,
+                    tofiledate=issue.created_at.isoformat(),
+                )
+            )
+            available_size = 500 - (len("```diff\n\n```"))
+            diff = truncate(diff, available_size)
+            content = f"```diff\n{diff}\n```"
+        elif old_issue_changes.title:
+            content = (
+                f'* Renamed from "{old_issue_changes.title.from_}" to "{issue.title}"'
```

Comment:
There's only one possible note now, and the single note case didn't have a bullet before: https://github.com/ghostty-org/discord-bot/blob/aa37d194c50235d14edef6d5159304c0e9f5f747/app/components/github_integration/webhooks/issues.py#L143-L144

```suggestion
                f'Renamed from "{old_issue_changes.title.from_}" to "{issue.title}"'
```